smartapp-core/README.md

```diff
@@ -169,6 +169,12 @@ To create your own implementation, you will need to:
      periods of over 30 days with no out-of-band activity, you will need to
      refresh the tokens on a schedule.
 
+### Reference Documentation
+
+Reference documentation can be found here:
+
+https://SmartThingsCommunity.github.io//smartapp-sdk-java/smartapp-core/javadoc/
```

Comment:
Extra `/` in the URL – surely this isn't required?

Reply:
It's not required but it redirects if you leave it off so I would rather leave it since it feels more accurate to me (and makes the page load a fraction of a second faster :-)).

Reply:
Okay I will trust you since you must have experimented with it. I can't seem to do that from this PR alone.